src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.Executor.cs

```diff
@@ -150,19 +150,16 @@ async Task<ImmutableDictionary<DiagnosticAnalyzer, DiagnosticAnalysisResult>> Me
                                 foreach (var document in project.Documents)
                                 {
                                     // don't analyze documents whose content failed to load
-                                    if (failedDocuments == null || !failedDocuments.Contains(document))
```

Comment:
we don't need to track of the content failed to load.  if it failed to load, it's just an empty string, which doesn't report diagnostics anyways.